dataclients/kubernetes/kube.go

```diff
@@ -351,13 +351,23 @@ func ParsePathMode(s string) (PathMode, error) {
 	}
 }
 
-func mapRoutes(r []*eskip.Route) map[string]*eskip.Route {
+func mapRoutes(r []*eskip.Route) (map[string]*eskip.Route, []*eskip.Route) {
```

Comment:
This would be called on each route poll cycle (every ~3s) so we risk spamming the logs.
Maybe we should ratelimit logging similar to `onPanicSometimes` in https://github.com/zalando/skipper/pull/2194

@szuecs WDYT?

Reply:
yes maybe, but then we can also argue this for all the "route update" info/error logs

Reply:
Maybe we should make it warning/alert that require fixing?

Reply:
Ok, lets keep it like that fro now.

> but then we can also argue this for all the "route update" info/error logs

lets do log ratelimiting for all logging in this module later then